src/routes/scope-selection/ScopeSelectionScreen.tsx

```diff
@@ -15,30 +19,60 @@ import { Discipline } from '../../constants/endpoints'
 import { useLoadDisciplines } from '../../hooks/useLoadDisciplines'
 import useReadSelectedProfessions from '../../hooks/useReadSelectedProfessions'
 import { RoutesParams } from '../../navigation/NavigationTypes'
-import { setSelectedProfessions } from '../../services/AsyncStorage'
-import { getLabels } from '../../services/helpers'
+import { pushSelectedProfession, setSelectedProfessions } from '../../services/AsyncStorage'
+import { getLabels, splitTextBySearchString } from '../../services/helpers'
 
-const DisciplineContainer = styled.View`
-  margin: 0 ${props => props.theme.spacings.sm};
+const HighlightContainer = styled.Text<{ disabled: boolean }>`
+  flex-direction: row;
+  color: ${props => (props.disabled ? props.theme.colors.disabled : props.theme.colors.black)};
 `
 
-const ButtonContainer = styled.View`
-  margin: ${props => props.theme.spacings.md} auto;
+const BoldText = styled.Text`
+  font-weight: bold;
 `
 
-const StyledText = styled(ContentSecondary)`
-  text-align: center;
+const StyledScrollView = styled.ScrollView`
+  background-color: ${props => props.theme.colors.background};
```

Comment:
This does not have any effect, does it? background has background color anyway?

Reply:
True 😅